commitrmnocb/outcome.go

```diff
@@ -171,21 +173,24 @@ func checkForReportTransmission(
 
 	if offRampUpdated {
 		return Outcome{
-			OutcomeType: ReportTransmitted,
+			OutcomeType:      ReportTransmitted,
+			LastPricesUpdate: previousOutcome.LastPricesUpdate,
```

Comment:
Instead of having this this common logic repeated in each of these, maybe raised to outside of the switch?
```
switch nextState {
    ...
}

// copy cache to next outcome if it isn't updated.
if (outcome.LastPricesUpdate == time.Time{}) {
    outcome.LastPricesUpdate = previousOutcome.LastPricesUpdate
}

p.lggr.Infow("Commit Plugin Outcome", "outcome", outcome, "oid", p.nodeID)
return outcome.Encode()
```